### ISSUE TITLE
[R] error in trying to download SAR2/human/USA dataset

### ISSUE THREAD

**Issue by valerioa** [ncbi/datasets#26]
Hello,

I'm trying to download the SAR2/human/USA dataset on an Linux/ARM64, for which there is no pre-compiled 'datasets' command line command.

I've run into an issue, using R:

```
library(ncbi.datasets)
api.virus_instance <- VirusApi$new()
api.virus_instance$VirusGenomeDownload(taxon="SARS2", host = "human", geo.location="USA", exclude.sequence=F)
```
returns 

```
Error in value[[3L]](cond) : Failed to deserialize response
```

and it downloads nothing.

**Comment by BradHolmes** [comment]
Hello,

Thank you for the feedback.  We will produce a Linux/ARM64 binary in the next week or two.  I'll close this issue once that is available.

You won't be able to download a zip file through R.  R doesn't know what to do with it.  Until we get the new CLI out, you can use the UI, or look into the python Jupyter notebooks at https://github.com/ncbi/datasets/tree/master/examples/jupyter

Sorry for the trouble!

**Comment by valerioa** [comment]
Thank you very much, Brad.
Unlike x86_64/amd64, arm64 platforms are extremely heterogeneous - there is no reference implementation, I think it would be hard to compile an executable that runs on all.
For R to do something with the download file, could this be working?

```
# at the top
library(readr)

# after resp is returned and query is successful
# resp <- self$apiClient$CallApi(url = paste0(self$apiClient$basePath, urlPath), ...)

raw_v = content(resp, as="raw")
write_file(raw_v, file="/tmp/tempfile")
file.rename("/tmp/tempfile", filename)
```

**Comment by valerioa** [comment]
And, truth to be told, I can do (on the arm64):

```
wget -Oncbi._datasets.zip https://api.ncbi.nlm.nih.gov/datasets/v1alpha/virus/taxon/SARS/genome/download?host=human\&geo_location=USA\&taxon=SARS
```

and get away with it

**Comment by BradHolmes** [comment]
Yes, wget that works just as well, I should have mentioned that!

As for R, to be honest, I am not expert enough to know if that syntax will work.  Let me know what you find out.

**Comment by valerioa** [comment]
Brad two comments:
1) I might have given you wrong information. An arm64 statically linked executable will run on any arm64 platform. The trick is statically linked. Look at this project https://github.com/mikefarah/yq . They release arm64 statically linked exes and they do work. I just tested it (on a Pine64). https://github.com/mikefarah/yq/releases/tag/3.4.1

2) The R code above works. I tested it. R will never deserialize on object that has not been serialized by another R process. And, it will never deserialize a zip file to a data.frame. I think the R api should save the file locally, for the download api (I think the python api does the same). For the download of data without sequences, R can transform a json file (containing an array of objects) info a dataframe, so that might be a possibility.
